plugins/module_utils/common/ep/nexus/api/api.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2024 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+__author__ = "Allen Robel"
+
+import logging
+
+from ..nexus import Nexus
+
+
+class Api(Nexus):
+    """
+    ## Nexus Dashboard enpoints - Nexus().Api()
+
+    ### Description
+    Common methods and properties for ND subclasses.
+
+    ### Path
+    ``/nexus/api``
```

Comment:
Do you know why the root of this endpoint name is `/nexus/`?

Reply:
This is the root for Nexus Dashboard endpoints, which is used by EppFederationMemberByName().  Though, this class is not yet being used (I figured it would be needed later for the OneManage stuff).

If you'd prefer, I could remove the Federation things for now and add them only if/when needed for OneManage.

Reply:
No need to remove.  Just curious